src/plugins/dashboard/public/services/dashboard_content_management/lib/find_dashboards.ts

```diff
@@ -83,19 +85,38 @@ export async function findDashboardById(
       references: cachedDashboard.item.references,
     };
   }
+
   /** Otherwise, fetch the dashboard from the content management client, add it to the cache, and return the result */
-  const response = await contentManagement.client
-    .get<DashboardCrudTypes['GetIn'], DashboardCrudTypes['GetOut']>({
+  try {
+    const response = await contentManagement.client.get<
+      DashboardCrudTypes['GetIn'],
+      DashboardCrudTypes['GetOut']
+    >({
```

Comment:
Had to refactor this - for some reason, the old code wasn't catching errors being thrown in `contentManagement.client.get`.